scripts/example_configs/femnist/run.sh

```diff
@@ -0,0 +1,8 @@
+# SHA
```

Comment:
A shell script in the folder of configuration files. Is it possible to organize the shell scripts better?